src/wp-includes/blocks.php

```diff
@@ -205,6 +205,10 @@ function register_block_style_handle( $metadata, $field_name, $index = 0 ) {
 		$style_handle = $style_handle[ $index ];
 	}
 
+	if ( wp_style_is( $style_handle, 'registered' ) ) {
```

Comment:
@spacedmonkey, could you please add a small description of this change so that others can understand it as well?